more_itertools/more.py

```diff
@@ -340,8 +340,18 @@ def one(iterable):
 def distinct_permutations(iterable):
```

Comment:
This one was missing an example. The `itertools` docs describe the number of elements returned for `permutations` and `combinations`, so I've added that here too.